packages/fast-components-styles-msft/src/media/index.ts

```diff
@@ -0,0 +1,16 @@
+import {IDesignSystem} from "../design-system";
+import {ComponentStyles} from "@microsoft/fast-jss-manager";
+import {IMediaClassNameContract} from "@microsoft/fast-components-class-name-contracts";
+
+const styles: ComponentStyles<IMediaClassNameContract, IDesignSystem> = {

```

Comment:
we'll want to move the image styles over prior to merging this in. Let me know if you guys all want to jump on a session via vscode and we can walk through actually doing one of these.